index.js

```diff
@@ -92,18 +96,24 @@ function createIdObject(row, importantEvents) {
     return {
         alternateId: row[alternateIdColumnNumber],
         github: row[githubIdColumnNumber],
-        contributions: importantEvents
+        contributions: importantEvents,
     };
 }
 
 function filterContributorByTime(idObject, moments) {
-    const startMoment = moments[0];
-    const endMoment = moments[1];
+    const startMoment = DateTime.fromISO(moments[0]);
+    const endMoment = DateTime.fromISO(moments[1]);
 
-    const timeWindow = moment.range([startMoment, endMoment]);
     for (let i = 0; i < idObject.contributions.length; i++) {
-        const momentOfContribution = moment.utc(idObject.contributions[i].created_at);
-        if (timeWindow.contains(momentOfContribution)) {
+        const momentOfContribution = DateTime.fromISO(idObject.contributions[i].created_at, {
+            zone: 'utc',
+            setZone: true,
```

Comment:
Are you sure setZone should be set true? The luxon docs are unfortunately not clear, but it sounds like setting it to true will use the parsed zone rather than the one being passed here. See: https://github.com/moment/luxon/issues/546

Reply:
I removed it, as I am pretty sure that we want the default behavior (to use the passed zone).